WPMedia/fusion-news-theme-blocks
# TMEDIA 252 nav list item transparency issue

## Description
When the sections menu has more items than can fit within the viewport, and the sections menu is closed additional white space is added to the bottom of the page.

## Jira Ticket
- [TMEDIA-252](https://arcpublishing.atlassian.net/browse/TMEDIA-252)

## Acceptance Criteria
Expected Behavior
1. No additional scrolling should happen when you reach the bottom of a page when
a. The sections menu is closed
b. The sections menu is closed and have more items than would fit within the viewport
2. When the sections menu is open
a. The sections menu should scroll to allow a user to see all menu items
b. User should be able to open multiple sub sections and still scroll to see all menu items

## Test Steps
- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout TMEDIA-252-nav-list-whitespace-issue`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/shared-styles,@wpmedia/header-nav-chain-block`
3. Go to http://localhost/homepage/?_website=the-gazette 
4. Click sidebar section
5. Open subsection
6. Scroll down the page and ensure background color of items is not transparent

## Effect Of Changes
### Before
![Screen Shot 2021-07-27 at 11 58 00](https://user-images.githubusercontent.com/5950956/127196124-a9c3be11-cad5-4042-80b2-0f24541edb7e.png)

### After
![Screen Shot 2021-07-27 at 11 57 25](https://user-images.githubusercontent.com/5950956/127196046-195fe278-6ff8-4b7b-85ad-51c358b200e1.png)

## Dependencies or Side Effects

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
